src/user_preferences.jl

```diff
@@ -189,9 +189,9 @@ function load_api_keys!()
     global AZURE_OPENAI_HOST
     AZURE_OPENAI_HOST = @load_preference("AZURE_OPENAI_HOST",
         default=get(ENV, "AZURE_OPENAI_HOST", ""))
-    global MISTRALAI_API_KEY
-    MISTRALAI_API_KEY = @load_preference("MISTRALAI_API_KEY",
-        default=get(ENV, "MISTRALAI_API_KEY", ""))
+    global MISTRAL_API_KEY
+    MISTRAL_API_KEY = @load_preference("MISTRAL_API_KEY",
+        default=get(ENV, "MISTRAL_API_KEY", ""))
```

Comment:
Could maybe add a warning here that `MISTRALAI_API_KEY is deprecated. Here's a clumsy way of doing that.

```suggestion
    global MISTRAL_API_KEY
    if !isempty(@load_preference("MISTRALAI_API_KEY",
        default=get(ENV, "MISTRALAI_API_KEY", "")))
        @warn "The MISTRALAI_API_KEY environment variable is deprecated. Use MISTRAL_API_KEY instead"
    end
    MISTRAL_API_KEY = @load_preference("MISTRAL_API_KEY",
        default=get(ENV, "MISTRAL_API_KEY", ""))
```